preprocessing/nextclade/tests/test_metadata_processing_functions.py

```diff
@@ -370,12 +370,12 @@
         ],
     ),
     Case(
-        name="non_ascii_authors",
+        name="non_latin_characters_authors",
         input_metadata={
-            "submissionId": "non_ascii_authors",
+            "submissionId": "non_latin_characters_authors",
             "name_required": "name",
             "ncbi_required_collection_date": "2022-11-01",
-            "authors": "Møller, Anäis; Pérez, José",
+            "authors": "Pérez, José; Bailley, François; 汉",
```

Comment:
In order to decode `Møller, Anäis` as ascii we would need to use the library `unidecode` - there we can decode many more characters - we will just have to check that they are still in the latin extended character set first (otherwise it will also decode non-latin characters and the authors warn this might not work) 

```
from unidecode import unidecode

def unidecode_latin_only(text):
    result = []
    for ch in text:
        codepoint = ord(ch)
        if 0x0000 <= codepoint <= 0x024F:
            # Apply unidecode only to Latin blocks
            result.append(unidecode(ch))
        else:
            raise ValueError(f"Not latin character {ch}")
    return ''.join(result)
  ```